src/geometry/manifold/instances/units_of_normed_algebra.lean

```diff
@@ -62,4 +172,54 @@ variables {𝕜 : Type*} [nontrivially_normed_field 𝕜] [normed_algebra 𝕜 R
 instance : smooth_manifold_with_corners 𝓘(𝕜, R) Rˣ :=
 open_embedding_coe.singleton_smooth_manifold_with_corners 𝓘(𝕜, R)
 
+lemma cont_mdiff_coe {m : with_top ℕ} : cont_mdiff 𝓘(𝕜, R) 𝓘(𝕜, R) m (coe : Rˣ → R) :=
+cont_mdiff_open_embedding 𝓘(𝕜, R) units.open_embedding_coe
```

Comment:
Could you please add a docstring to this lemma?  It will be an important part of how people interact with this Lie group.  Please also add the following companion lemma (also with a docstring) and use it instead of `cont_mdiff.of_comp_open_embedding` below so that readers have an example of the basic process.
```lean
lemma cont_mdiff.of_comp_units_coe
  {E : Type*} [normed_add_comm_group E] [normed_space 𝕜 E]
  {H : Type*} [topological_space H] {I : model_with_corners 𝕜 E H}
  {M : Type*} [topological_space M] [charted_space H M]
  {n : with_top ℕ}
  {f : M → Rˣ} (hf : cont_mdiff I 𝓘(𝕜, R) n ((coe : Rˣ → R) ∘ f)) :
  cont_mdiff I 𝓘(𝕜, R) n f :=
cont_mdiff.of_comp_open_embedding units.open_embedding_coe hf
```